src/utils/i18n.ts

```diff
@@ -334,3 +334,12 @@ export function lookupPhrase(options: LookupOptions): string {
   // Fallback to the key.
   return key;
 }
+
+export function isValidLanguageTag(languageTag: string): boolean {
+  try {
+    new LanguageTag(languageTag);
+    return true;
+  } catch (e) {
+    return false;
+  }
+}
```

Comment:
I kind of expected the logic that detects if the string is valid to be pulled out of the `LanguageTag` and put here. And then instantiating a `LanguageTag` would throw an error if `isValidLanguageTag` returned false.